src/test/java/org/mockito/internal/matchers/text/ValuePrinterTest.java

```diff
@@ -14,7 +16,14 @@ public void prints_values() throws Exception {
         assertEquals("null", print(null));
         assertEquals("\"str\"", print("str"));
         assertEquals("\"x\ny\"", print("x\ny"));
+        assertEquals("3", print(3));
+        assertEquals("3L", print(3L));
         assertEquals("[1, 2]", print(new int[]{1, 2}));
+        assertEquals("{\"hoge\"=2L}", print(new HashMap<String, Object>() {
+            {
+                put("hoge", 2L);
+            }
+        }));
```

Comment:
Probably for multiple values it would be good to add some white-space around the `=` and maybe put them on a new-line.